files/cleanup-registry.py

```diff
@@ -65,27 +66,28 @@ def get_organization(giteaApi):
             orgs.append(elt['username'])
         return orgs
     except requests.exceptions.HTTPError as err:
-        logging.error('Error %s when trying to get the organization: %s' % (err.response.status_code, err.response.text))
+        logging.error(f'Error {err.response.status_code} when trying to get the organization: {err.response.text}')
 
 def get_artifacts(giteaApi, org):
-    logging.info('Querrying org %s to list all artifacts' % org)
+    logging.info(f'Querrying org {org} to list all artifacts')
     try:
         arts=[]
-        res=giteaApi._get('packages/%s' %  org)
+        res=giteaApi._get(f'packages/{org}')
         for elt in res.json():
             if elt['type'] == 'container':
-                arts.append(Artifact(elt['name'], elt['owner'], elt['created_at'], elt['version']))
+                arts.append(Artifact(elt['name'], elt['owner']['full_name'], elt['created_at'], elt['version']))
         return arts
     except requests.exceptions.HTTPError as err:
-        logging.error('Error %s when trying to get all the artifact of %s: %s' % (err.response.status_code, org, err.response.text))
+        logging.error(f'Error {err.response.status_code} when trying to get all the artifact of {org}: {err.response.text}')
         raise err
 
 def delete_artifact(giteaApi, art):
-    logging.info('Deleting artifact %s of %s' % (art.name, art.owner))
+    logging.info(f'Deleting artifact {art.name} of {art.owner}')
     try:
-        res=giteaApi._del('packages/%s/container/%s/%s' % (art.owner, art.name, art.version))
+        path = f'packages/{art.owner}/container/{quote(art.name, safe='')}/{art.version}'
```

Comment:
```suggestion
        path = f'packages/{art.owner}/container/{quote(art.name, safe="")}/{art.version}'
```

This quote can cause issues like : 

```
Mar 11 00:00:05 node-01.he-eu-fsn1.ci.gitea systemd[1]: Starting Timer to perform clean...
Mar 11 00:00:05 node-01.he-eu-fsn1.ci.gitea cleanup-registry.py[1866291]:   File "/docker/gitea/cleanup-registry.py", line 87
Mar 11 00:00:05 node-01.he-eu-fsn1.ci.gitea cleanup-registry.py[1866291]:     path = f'packages/{art.owner}/container/{quote(art.name, safe='')}/{art.version}'
```